Documentation/CRDs/Block-Storage/ceph-block-pool-rados-namespace-crd.md

```diff
@@ -36,6 +36,11 @@ metadata:
 spec:
   # The name of the CephBlockPool CR where the namespace is created.
   blockPoolName: replicapool
+  mirroring:
```

Comment:
As in the other comment, let's create a separate example that is specifically for mirroring. This keeps the main example simple, and we can also mention in the separate example what the min Ceph version is for that feature.